docs/developer/extending/plugins/openid_connect.mdx

```diff
@@ -0,0 +1,262 @@
+    Saleor ->>+ OpenIDConnectPlugin: authentication backend calls <br>Plugin.authenticate_user
+    OpenIDConnectPlugin ->> OpenIDConnectPlugin: decode and validate token
+    OpenIDConnectPlugin ->> OpenIDConnectPlugin: get user
+    OpenIDConnectPlugin ->> OpenIDConnectPlugin: assign permissions <br>from token
+    OpenIDConnectPlugin ->>- Saleor: user authenticated
+    Saleor ->>- Web: requested resources
+`}
+/>
+
+## OAuth permissions
+Saleor can use OAuth permissions assigned to a user. It maps internal permissions to OAuth scopes and attaches 
+it to the authorization request.
+
+
+### OAuth permission scopes
+| OAuth scope name           | Saleor Name                 | Description                                                                                                                      |
+| -------------------------- | --------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| saleor:manage_staff        | MANAGE_STAFF                | Access to staff users data                                                                                                       |
+| saleor:manage_apps         | MANAGE_APPS                 | Manage apps                                                                                                                      |
+| saleor:manage_users        | MANAGE_USERS                | Access to customers data                                                                                                         |
+| saleor:manage_discounts    | MANAGE_DISCOUNTS            | Manage discounts                                                                                                                 |
+| saleor:manage_plugins      | MANAGE_PLUGINS              | Manage plugins                                                                                                                   |
+| saleor:manage_gift_card    | MANAGE_GIFT_CARD            | Manage gift cards                                                                                                                |
+| saleor:manage_menus        | MANAGE_MENUS                | Manage the structure of menus                                                                                                    |
+| saleor:manage_orders       | MANAGE_ORDERS               | Access to orders data                                                                                                            |
+| saleor:manage_pages        | MANAGE_PAGES                | Manage pages                                                                                                                     |
+| saleor:manage_product      | MANAGE_PRODUCTS             | Manage products                                                                                                                  |
```

Comment:
We've split `MANAGE_PRODUCTS` into: `MANAGE_PRODUCTS` and `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES`